webservices/rest.py

```diff
@@ -219,6 +219,16 @@ def add_aggregate_resource(api, view, schedule, label):
     '/committee/<committee_id>/filings/',
     '/candidate/<candidate_id>/filings/',
 )
+
+api.add_resource(
+    filings.EFilingSummaryView,
+    '/efile/presidential/<string:form_type>/',
```

Comment:
+1 to @LindsayYoung 's suggestion. I like the idea of having the efiling version of an endpoint begin with `efile/` and then have everything else be as similar as possible after that.